prov/efa/src/rdm/rxr_ep.c

```diff
@@ -1154,7 +1151,12 @@ static
 int rxr_ep_set_sendrecv_in_order_aligned_128_bytes(struct rxr_ep *ep,
 						   bool sendrecv_in_order_aligned_128_bytes)
 {
-	if (sendrecv_in_order_aligned_128_bytes)
+	/*
+	 * RDMA read is used to copy data from host bounce buffer to the
+	 * application buffer on device
+	 */
+	if (sendrecv_in_order_aligned_128_bytes &&
+	    !efa_base_ep_support_op_in_order_aligned_128_bytes(&ep->base_ep, IBV_WR_RDMA_READ))
```

Comment:
I think this should be the last commit of this PR.

Reply:
Updated.